src/components/Table/Table.qa.stories.tsx

```diff
@@ -106,45 +113,29 @@ type BeamChildRow = { kind: "child"; id: string } & BeamBudgetData;
 type BeamNestedRow = BeamTotalsRow | HeaderRow | BeamParentRow | BeamChildRow;
 
 export function NestedFixed() {
-  const totalsRowStyles = useMemo(() => ({ totals: { cellCss: beamTotalsRowStyle } }), []);
-  const rowStyles = useMemo(() => ({ parent: { cellCss: beamGroupRowStyle } }), []);
   return (
     <div css={Css.mw("fit-content").$}>
+      <GridTable<BeamNestedRow> style={beamTotalsFixedStyle} columns={beamNestedColumns} rows={beamTotalsRows} />
```

Comment:
@Levi-Huynh this story is a good example of what we'll need to follow for the commitments page...

In particular, for the header we used to embed totals directly into the headers row, but now they want to pull headers out into their own like "pre-header" row.

Which is fine, but just b/c of the way our table internals work, we have to do this thing of "okay, that means two `GridTable`s, one with just the total row, and then one with everything else" (and wrapped in a fit-content div so they line up).

I think eventually we'll either fix this, so the total row can be included directly in the GridTable, or we'll make like a `TableWithTotal` component that just wraps the two separate GridTables internally...

But, for now, this isn't too bad to just copy/paste...